packages/skeleton/src/lib/components/AppShell/AppShell.svelte

```diff
@@ -9,7 +9,11 @@
 	 */
 
 	// Types
-	import type { CssClasses } from '../../index.js';
+	import type { CssClasses, SvelteEvent } from '../../index.js';
+
+	interface $$Events {
+		scroll: SvelteEvent<Event, HTMLDivElement>;
```

Comment:
It's better to define the event type for what it is, rather than a generic Event
```suggestion
		scroll: SvelteEvent<UIEvent, HTMLDivElement>;
```

Reply:
I missed that, thanks